addons/mail/static/src/owl/components/composer/composer.js

```diff
@@ -365,6 +366,10 @@ Composer.defaultProps = {
 };
 
 Composer.props = {
+    mentionedPartners: {
+        type: Array,
+        element: Object,
```

Comment:
I think we usually try to specify objects with "shape".

Reply:
https://github.com/odoo/owl/blob/master/doc/reference/props_validation.md

Reply:
Done